src/utils/generateJwt.ts

```diff
@@ -0,0 +1,19 @@
+import jwt from '@tsndr/cloudflare-worker-jwt';
+import { env } from '../types/global.types';
+
+export const generateJwt = async (env: env) => {
```

Comment:
exp claim in the JWT token should be in the future, not in the past. The current time is 2023-07-24 11:41:54 PST, so the exp claim should be set to a value greater than that. For example, you could set it to Math.floor(Date.now() / 1000) + 300

Reply:
We get the server allocated to us only for 10ms. And as per the recent usage we have found this timestamp suits us.